launcher/minecraft/skins/SkinModel.cpp

```diff
@@ -23,10 +23,17 @@
 #include "FileSystem.h"
 #include "Json.h"
 
-static QImage improveSkin(const QImage& skin)
+static QImage improveSkin(const QImage& _skin)
 {
+    auto skin = _skin;
```

Comment:
Am I missing something here? What's up with this binding?

Reply:
Yes you are missing the part where I pass the `_skin` as a const reference, as I did not want to copy the image when the function was called.
Should I just remove the const reference from the function argument?